platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/ArrowDirection.java

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.mapboxsdk.annotations;
+
+public enum ArrowDirection {
```

Comment:
I personally prefer using `@IntDef` instead of an int-enum as enums have a unnecessary large memory footprint for the functionality that they provide. An `@IntDef` enforces using certain values through lint in the same manner as type safety is assured with enums. Happy to discuss this in more detail!